e2e/pages/k8s-page.ts

```diff
@@ -0,0 +1,18 @@
+import { Page, Locator } from 'playwright';
+import { expect } from '@playwright/test';
+
+export class K8sPage {
+    readonly page: Page;
+    readonly k8sMemorySliderSelector: Locator;
+    readonly k8sCpuSliderSelector: Locator;
+    readonly k8sPortSelector: Locator;
+    readonly k8sResetBtn: Locator;
```

Comment:
This doesn't look too long; would `k8sResetButton` be okay?

Reply:
Sure thing, thanks for calling this out.